ivy/functional/ivy/linear_algebra.py

```diff
@@ -612,6 +612,94 @@ def svdvals(
         length ``K``, where K = min(M, N). The values are sorted in descending order by
         magnitude.
 
+    Functional Examples
+    -------------------
+
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.random_normal(shape = (8, 6))
+    >>> S = ivy.svdvals(x)
+    >>> print(S.shape)
+    (6,)
+
+    Compare the singular value S by ivy.svdvals() with the result by ivy.svd().
+
+    >>> _, SS, _ = ivy.svd(x)
+    >>> pring(SS.shape)
+    (6,)
+    >>> error = (SS - S).abs()
+    >>> print(error)
+    ivy.array([0.00e+00, 2.38e-07, 2.38e-07, 2.38e-07, 0.00e+00, 1.19e-07])
+
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([[1.0, 2.0, 3.0], [2.0, 3.0, 4.0],\
+                              [2.0, 1.0, 3.0], [3.0, 4.0, 5.0]])
+    >>> print(x.shape)
+    (4, 3)
+    >>> S = ivy.svdvals(x)
+    >>> print(S)
+    ivy.array([10.3  ,  1.16 ,  0.615])
+
+    Compare the singular value S by ivy.svdvals() with the result by ivy.svd().
+    >>> _, SS, _ = ivy.svd(x)
+    >>> print(SS)
+    ivy.array([10.3  ,  1.16 ,  0.615])
```

Comment:
Can you please leave a single newline after each output of the print statement? It is needed as `ivy_tests/test_docstrings.py` is set up in such a way that it expects the output of a print statement to end at '\n'.

Reply:
@MarShaikh Sure, it is done! Thanks for your review and advice!